src/main/java/org/eclipse/yasson/internal/serializer/ShortArraySerializer.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2016, 2017 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
why there is a copyright year change here while there are no other changes in this class?

Reply:
This class was modified in one of the commits. This change was later reverted, but it will remain in the history so it was modified in 2019.